module/Photo/src/Photo/Service/Album.php

```diff
@@ -62,6 +62,45 @@ public function getAlbums($album = null, $start = 0, $maxResults = null)
     }
 
     /**
+     * Returns all albums for a given year.
+     * Example: When supplied 2010, this would return all albums from 2010/2011
+     *
+     * @param $year integer the year in which the albums have been created
+     *
+     * @return array of \Photo\Model\Albums
+     */
+    public function getAlbumsByYear($year) {
+        // A GEWIS year starts July 1st
+        $start = \DateTime::createFromFormat('Y-m-d H:i:s', "$year-07-01 0:00:00");
```

Comment:
Kun je de 07 in een constante zetten, (die ook bij een van mijn vorige reacties gebruikt word)

Daarbij, waarom niet van Y-m-d als formaat een datum creeren